### DISCUSSION TITLE
humour, or randomised words which don't look even slightly believable. If you

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#277] [Q&A] [answered]
ook even slightly believable. If you

**Comment by mertcangokgoz** [comment, marked as answer]
It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
